src/app/waste-water/activated-sludge-form/activated-sludge-form.component.html

```diff
@@ -26,10 +26,54 @@ <h3>Activated Sludge Data</h3>
             </span>
         </div>
         <!-- So -->
+
+        <div class="form-group" *ngIf="!this.form.controls.isUserDefinedSo.value"
+        [ngClass]="{'indicate-different': activatedSludgeDifferent.influentCBODBefore, 'invalid': form.controls.influentCBODBefore.invalid}">
+            <label>Influent CBOD<sub>5</sub> Before</label>
```

Comment:
The two new fields still don't use disabled styling (greyed out) when they aren't selected. Use the [readonly]="!selected" like the other fields on this form